### PR TITLE
Cms-Kit: Allow to write global css/js 

### LINKED ISSUE #11690: Allow to write global css/js for CMS Kit module

### PR DESCRIPTION
**Pull request by yekalkan**
Resolves https://github.com/abpframework/abp/issues/11690

image

### CODE REVIEW

**Comment by hikalkan** [comment]
@yekalkan please help to @gizemmutukurt to test this.

File: modules/cms-kit/src/Volo.CmsKit.Admin.Application.Contracts/Volo/CmsKit/Permissions/CmsKitAdminPermissions.cs [hunk 1]
```diff
@@ -60,4 +60,9 @@ public static class Menus
         public const string Update = Default + ".Update";
         public const string Delete = Default + ".Delete";
     }
+
+    public static class GlobalResources
+    {
+        public const string Default = GroupName + ".Menus";
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.Admin.Application.Contracts/Volo/CmsKit/Permissions/CmsKitAdminPermissions.cs]:
```suggestion
        public const string Default = GroupName + ".GlobalResources";
```

File: modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi.Client/ClientProxies/GlobalResourcesAdminClientProxy.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿// This file is part of GlobalResourcesAdminClientProxy, you can customize it here
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi.Client/ClientProxies/GlobalResourcesAdminClientProxy.cs]: Is this file accidently added? There is `GlobalResourceAdminClientProxy` above.

File: modules/cms-kit/src/Volo.CmsKit.EntityFrameworkCore/Volo/CmsKit/GlobalResources/EfCoreGlobalResourceRepository.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+namespace Volo.CmsKit.GlobalResources;
+
+public class EfCoreGlobalResourceRepository: EfCoreRepository<ICmsKitDbContext, GlobalResource, Guid>, IGlobalResourceRepository
+{
+    public EfCoreGlobalResourceRepository(IDbContextProvider<ICmsKitDbContext> dbContextProvider) : base(dbContextProvider)
+    {
+        
+    }
+
+    public Task<GlobalResource> FindByName(string name,
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.EntityFrameworkCore/Volo/CmsKit/GlobalResources/EfCoreGlobalResourceRepository.cs]:
```suggestion
    public Task<GlobalResource> FindByNameAsync(string name,
```

File: modules/cms-kit/src/Volo.CmsKit.MongoDB/Volo/CmsKit/MongoDB/GlobalResources/MongoGlobalResourceRepository.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace Volo.CmsKit.MongoDB.GlobalResources;
+
+public class MongoGlobalResourceRepository: MongoDbRepository<ICmsKitMongoDbContext, GlobalResource, Guid>, IGlobalResourceRepository
+{
+    public MongoGlobalResourceRepository(IMongoDbContextProvider<ICmsKitMongoDbContext> dbContextProvider) : base(dbContextProvider)
+    {
+    }
+
+    public Task<GlobalResource> FindByName(string name,
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.MongoDB/Volo/CmsKit/MongoDB/GlobalResources/MongoGlobalResourceRepository.cs]:
```suggestion
    public Task<GlobalResource> FindByNameAsync(string name,
```

File: modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Script/Default.cshtml [hunk 5]
```diff
@@ -0,0 +1 @@
+﻿<script src="/global-resources/script"></script>
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Script/Default.cshtml]: `/global-resources/script` is too common. change to `/cms-kit/global-resources/script`. Also, I didn't see, which controller matches this URL?

File: modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Style/Default.cshtml [hunk 6]
```diff
@@ -0,0 +1 @@
+﻿<link rel="stylesheet" href="/global-resources/style"/>
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Style/Default.cshtml]: `/global-resources/style` is too common. change to `/cms-kit/global-resources/style` .

File: modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/GlobalResources/index.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+$(function (){
+    var l = abp.localization.getResource("CmsKit");
+    
+    var service = volo.cmsKit.admin.globalResources.globalResourceAdmin;
+
+    $('#SaveResourcesButton').on('click','',function(){
```
Comment (hikalkan) [inline modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/GlobalResources/index.js:6]: If we have a single save button, why we have separate service methods (`setGlobalStyle` and `setGlobalScript`), and we have only one method for get.
I think this is not consistent. Unify these two methods into a single method and make a single AJAX call.

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Review by hikalkan — APPROVED** [review]

**Comment by hikalkan** [comment]
@gizemmutukurt that's OK for me. Please you visually test and merge.

**Review by gizemmutukurt — APPROVED** [review]

**Comment by gizemmutukurt** [comment]
Test is OK.

**Pull request merged**